runtime/gc_base/GCExtensions.cpp

```diff
@@ -241,25 +240,23 @@ MM_GCExtensions::computeDefaultMaxHeap(MM_EnvironmentBase *env)
  * internal requirements given that one compilation thread can use up to 256M.
  */
 #define OPENJ9_IN_CGROUP_NATIVE_FOOTPRINT_EXCLUDING_HEAP ((U_64)512 * 1024 * 1024)
-
+		uint64_t cgroupMemory = 0;
```

Comment:
There is no usage of cgroupMemory in this code. Why do we need a call of omrsysinfo_cgroup_get_memlimit() here?

Reply:
call to `omrsysinfo_cgroup_get_memlimit()` would tell us if there is any memory limit imposed by cgroup in which case we set the heap to large size.

Reply:
ok, I see. RC is still be in use